labs/pipelines-review/pipeline.yaml

```
@@ -0,0 +1,59 @@
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: maven-java-pipeline
+spec:
+  workspaces:
+    - name: shared
+    - name: maven_config
+  params:
+    - name: GIT_REPO
+      type: string
+      default: "https://git.ocp4.example.com/developer/DO288-apps"
+    - name: GIT_REVISION
+      type: string
+      default: "master"
+    - name: MVN_APP_PATH
+      type: string
+      default: "builds-applications/apps/vertx-site"
+    - name: DEPLOY_ARTIFACT_NAME
+      type: string
+      default: "target/vertx-site-1.0.0-SNAPSHOT-fat.jar"
+    - name: DEPLOY_APP_NAME
+      type: string
+      default: "vertx-site"
+
+  tasks:
+    - name: clone-repository
+      taskRef:
+        # TODO
+      params:
+        # TODO
+        - name: deleteExisting
+          value: 'true'
+        - name: sslVerify
+          value: 'false'
+      workspaces:
+        # TODO
+
+    - name: build
+      taskRef:
+        # TODO
+      params:
+        # TODO
+      workspaces:
+        # TODO
+
+    - name: oc-deploy
+      taskRef:
+        # TODO
+      workspaces:
+        # TODO
+      params:
+        # TODO
+
+  - name: skopeo-copy
+    taskRef:
+      # TODO
+    params:
+      # TODO
```

This would fix indentation
```suggestion
    - name: skopeo-copy
      taskRef:
        # TODO
      params:
        # TODO
```